07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(set(genes) - set(coefs.keys()))
</pre><p>You could add an <code>assert</code> here if you already know that the set difference should be empty:</p><pre class="ql-syntax" spellcheck="false">assert (len(set(genes) - set(coefs.keys())))) == 0
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>